### ISSUE TITLE
Manifest URL no longer installs in foundry

### ISSUE THREAD

**Issue by asacolips** [asacolips-projects/pbta#43]
In GitLab by @sethblevins on Jun 15, 2021, 22:51

## Summary

Installing with the link gives this error. 
![Untitled](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/cdcc3144933b983520fcbd071a98076a/Untitled.png)
## Labels (do-not-change)

/cc @Asacolips

**Comment by asacolips** [comment]
In GitLab by @Asacolips on Jun 16, 2021, 08:49

Thanks for the report @sethblevins! This should be resolved now.

**Comment by asacolips** [comment]
In GitLab by @sethblevins on Jun 19, 2021, 16:56

Still get the exact same problem with Foundry. Unless the manifest url has change. I just tried the one on the front page. 
https://gitlab.com/asacolips-projects/foundry-mods/pbta/-/jobs/artifacts/beta/raw/system.json?job=build-beta

**Comment by asacolips** [comment]
In GitLab by @Asacolips on Jun 21, 2021, 08:59

@sethblevins what URL are you trying to use that's giving the error? The issue I had resolved was that there wasn't an active build artifact, so I triggered a new one, at which point the URLs started working for me. I've tested the one you listed in your comment above and it was able to install the beta for me. If that's the URL you're currently using and it's still not working, you could try this URL which is part of the repo itself and not an artifact: https://gitlab.com/asacolips-projects/foundry-mods/pbta/-/raw/beta/src/system.json

**Comment by asacolips** [comment]
In GitLab by @sethblevins on Jun 21, 2021, 23:35

That second link worked, but the one I posted still does not work. It gives the same error I show in the pic.

**Comment by asacolips** [comment]
In GitLab by @Asacolips on Jun 22, 2021, 17:02

@sethblevins that's odd, it's worked for me whenever I've tried installing locally, on my cloud server, and on the Forge. It could potentially be a browser thing, as the generated URL (the one in your comment) has different headers than the raw version of the manifest in the repo. The link that I posted will continue to be available if you want to keep it noted somewhere, and I'll update the next beta version to switch to the link I posted instead.
